tmt/utils.py

```diff
@@ -373,6 +377,14 @@ def last_run(self, workdir: Path) -> None:
             raise GeneralError(
                 f"Unable to save last run '{self.path}'.\n{error}")
 
+    @cached_property
+    def fmf_tree(self) -> fmf.Tree:
+        """ Return the configuration tree """
+        try:
+            return fmf.Tree(self.path)
+        except fmf.utils.RootError:
+            raise MetadataError(f"Config tree not found in '{self.path}'.")
```

Comment:
I miss `from error`, is there a reason why it shouldn't be linked to this exception?

Reply:
Sure, should be there. Fixed in 842df674.